{
	"title": "References Multiple Datasets throws InsightError",
	"input": {
		"WHERE": {},
		"OPTIONS": {
			"COLUMNS": [
				"sections2_dept",
				"sections_avg"
			]
		}
	},
	"errorExpected": true,
	"expected": "InsightError"
}